include/domain_bridge/parse_domain_bridge_yaml_config.hpp

```diff
@@ -102,6 +103,29 @@ class YamlParsingError : public std::runtime_error
 DOMAIN_BRIDGE_PUBLIC
 DomainBridgeConfig parse_domain_bridge_yaml_config(std::filesystem::path file_path);
 
+/// Create a DomainBridgeConfig object from some YAML files.
+/**
+ * If the passed files have overlapping settings, the one specified in a later file will apply.
+ *
+ * See \ref parse_domain_bridge_yaml_config() for more details.
+ */
+DOMAIN_BRIDGE_PUBLIC
+DomainBridgeConfig
+parse_domain_bridge_yaml_configs(std::vector<std::filesystem::path> file_paths);
```

Comment:
```suggestion
parse_domain_bridge_yaml_configs(const std::vector<std::filesystem::path> & file_paths);
```

Reply:
Thanks, I was planning to make that change but I forgot.
Fixed in https://github.com/ros2/domain_bridge/pull/64/commits/7c94e5e12de969d25c287f849ec61417216b26cc.